src/lib.rs

```diff
@@ -1115,36 +1120,36 @@ mod tests {
     test_pp_digest_with::<PallasEngine, _, _, EE<_>, EE<_>>(
       &TrivialCircuit::default(),
       &TrivialCircuit::default(),
-      &expect!["582db42439c0dcfc60d24b023ab83d81d97382ac2efa883c6f23147345efeb01"],
+      &expect!["e5a6a85b77f3fb958b69722a5a21bf656fd21a6b5a012708a4b086b6be6d2b03"],
```

Comment:
Why do we have changed test vectors?

Reply:
The changes to the R1CS coefficient matrix in src/bellpepper/r1cs.rs change the `R1CSWithArity` (ex-`R1CSShape`), and therefore the public parameters, and therefore their hash.